src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Distinct.txt

```diff
@@ -0,0 +1,92 @@
+// Blank table
+>> Distinct(If(1<0,[1,2]), true)
+Blank()
+
+// Empty table
```

Comment:
It looks like PowerApps does allow record comparison!

CountRows(Distinct(Table({x:1,y:2},{x:1,y:2}), ThisRecord)) // 1

First(Distinct(Table({x:1,y:2},{x:10,y:2}), ThisRecord)).Result.x // 1

Reply:
Added test demoing we only support primitive types.